web/keyApi.go

```diff
@@ -0,0 +1,77 @@
+package web
+
+import (
+	"integrand/services"
+	"log/slog"
+	"net/http"
+	"regexp"
+)
+
+var (
+	keyAllApi    = regexp.MustCompile(`^\/api\/v1\/apikey[\/]*$`)
+	keySingleApi = regexp.MustCompile(`^\/api\/v1\/apikey\/(.*)$`)
+)
+
+type keyAPI struct{}
+
+func (ka *keyAPI) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	switch {
+	case keySingleApi.MatchString(r.URL.Path):
+		enableCors(&w)
```

Comment:
Let's go ahead and disable cors for both routes. Remove this `enableCors()` on bother routes